bedrock/firefox/templates/firefox/desktop/index.html

```diff
@@ -21,7 +21,7 @@
   {% endif %}
 {% endblock %}
 
-{% block site_css %}
+{% block page_css %}
```

Comment:
Yep, this makes sense. We'll clean it up in [bug 1186605](https://bugzilla.mozilla.org/show_bug.cgi?id=1186605).